src/raftstore/store/worker/apply.rs

```diff
@@ -1449,14 +1442,16 @@ impl Runner {
             delegates: delegates,
             notifier: notifier,
             sync_log: sync_log,
+            tag: format!("[store {}]", store.store_id()),
```

Comment:
Could you aviod allocation here ?

Reply:
There is only one allocation, no need to optimize.